CarpetCleaning/StoredProcedures/Dbo.Loadexceldata.sql

```diff
@@ -0,0 +1,93 @@
+CREATE PROCEDURE dbo.LoadExcelData
+AS
+BEGIN
+
+WHILE ((SELECT COUNT(*) FROM ExcelData WHERE LoadDataNotes = 'Passed Validation') > 0)
+BEGIN
+
+	DECLARE @ThisRowId INT, 
```

Comment:
Fix the indentation on these lists, sometimes it's a difference between using indents and using spaces.  You might have used tab, but instead you might have to just add a bunch of spaces to get these at the same indentation level.